lldb/test/CMakeLists.txt

```diff
@@ -29,6 +29,22 @@ if(LLDB_ENFORCE_STRICT_TEST_REQUIREMENTS)
         "`LLDB_ENFORCE_STRICT_TEST_REQUIREMENTS=OFF`")
     endif()
   endforeach()
+
+  # On Windows make is not part of the MSYS tools that llvm-lit takes care of
+  if(LLDB_TEST_MAKE)
+    set(LLDB_DEFAULT_TEST_MAKE ${LLDB_TEST_MAKE})
+  else()
+    find_program(LLDB_DEFAULT_TEST_MAKE make)
+    if(LLDB_DEFAULT_TEST_MAKE)
+      message(STATUS "Found make: ${LLDB_DEFAULT_TEST_MAKE}")
+    else()
+      message(STATUS "Not found: make")
+      message(SEND_ERROR
```

Comment:
TIL about `SEND_ERROR`. I think it would be nice to use that for the rest of the strict requirements, but obviously that's outside the scope of this PR.

Reply:
https://github.com/llvm/llvm-project/pull/111531

Reply:
https://github.com/llvm/llvm-project/pull/111729